Rubberduck.RibbonDispatcher/ExcelRibbon/CustomButtonsViewModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+using System.Windows.Forms;
+using Microsoft.Office.Core;
+
+namespace RubberDuck.RibbonDispatcher {
+    using static RibbonControlSize;
+
+    internal class CustomButtonsViewModel {
```

Comment:
So... we add a demo Ribbon that does nothing useful and that the user can't customize without editing the C# source and recompiling? Not sure I'm following...